test/unit/vl-map-layer.test.html

```diff
@@ -63,7 +76,10 @@
     </template>
   </test-fixture>
 
-  <script>
+  <script type="module">
+    import { awaitUntil } from '/node_modules/vl-ui-core/vl-core.js';
+    import { isEqual } from '/node_modules/lodash-es/lodash.js';
```

Comment:
Dependency wordt alleen gebruikt om te controleren of arrays gelijk zijn? Lijkt me overkill? De dependency kan vervangen worden door bijvoorbeeld:

```JavaScript
[0, 1] == [0, 1]; // false
JSON.stringify([0, 1]) == JSON.stringify([0, 1]); // true
equalCoordinates([0, 1], [0, 1]); // true

function equalCoordinates(c1, c2) {
   return JSON.stringify(c1) == JSON.stringify(c2);
}
```

Reply:
Klopt, hadden we toen besproken en was een twijfelgeval. Maar uiteindelijk was er toen voor gekozen om het toch te gebruiken omdat de library reeds een transitive dependency is, en dus sowieso wordt binnengetrokken.
Kan natuurlijk nog terug aangepast worden, maar we gaan de library toch blijven binnentrekken.